components/standings/commons/starcraft_starcraft2/standings_storage_starcraft_legacy.lua

```diff
@@ -0,0 +1,140 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Standings/Storage/StarcraftLegacy
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+
+local Opponent = Lua.import('Module:Opponent', {requireDevIfEnabled = true})
+local StandingsStorage = Lua.import('Module:Standings/Storage', {requireDevIfEnabled = true})
+
+local LEAGUE_TYPE = 'league'
+local TYPE_FROM_NUMBER = {
+	Opponent.solo,
+	Opponent.duo,
+	Opponent.trio,
+	Opponent.quad,
+}
+local INVALID_OPPONENT_CATEGORY = '[[Category:Pages with invalid opponent '
+	.. 'parsing in legacy group tables]]'
+
+local Wrapper = {}
+
+function Wrapper.table(frame)
+	local args = Arguments.getArgs(frame)
+	-- option to disable storage in case of transclusions etc
+	if not Logic.readBool(Logic.emptyOr(args.store, true)) then
```

Comment:
How would you be able to set this on transclusions?

Reply:
```python pwb.py replace -transcludes:"TemplateArchive:GroupTableStart" -transcludes:"TemplateArchive:GroupTableSlot" -ns:0 -summary:"Storage handling for transcluded groups" "GroupTableStart" "GroupTableStart<includeonly>|store=false</includeonly>" "GroupTableSlot" "GroupTableSlot<includeonly>|store=false</includeonly>"```